### ISSUE TITLE
mac os 10.14, mgba 0.8-5373-357e2e2d: exiting full screen makes blank empty window

### ISSUE THREAD

**Issue by EzioisAwesome56** [mgba-emu/mgba#1212]
bug tested and present on mgba versions 0.8-5373-357e2e2d (lasted dev build as of 10-8-2018) and mgba 0.7-b1, does not happen on mgba 0.6.3

when entering fullscreen mode and exiting, a weird, blank white window with no buttons or close button opens behind it, and only goes away by closing mgba.

continuing to enter and exit fullscreen will keep making more and more of these windows. Happens in all video modes.

[Video of bug](https://youtu.be/bhb4kdXE3T8)

**Comment by endrift** [comment]
I'm seeing some of these symptoms happening on Steam Deck too, which makes me even more curious what's going on.

**Comment by endrift** [comment]
I make a change recently that may have fixed this. Can you test a recent dev build to see if it still happens?

**Comment by EzioisAwesome56** [comment]
> I make a change recently that may have fixed this. Can you test a recent dev build to see if it still happens?

sorry for delay, didnt notice this comment. Mind you, the computer/macos version has changed since i filed that bug report, but atleast with mGBA 0.11-8854-cef7dde50 on Intel MacOS Sequoia 15.5, this bug no longer occurs. I saw there were legacy builds too if you want testing on MacOS 10.9 as well

**Comment by endrift** [comment]
Nah, the legacy builds are discontinued. I'm just gonna call this done unless someone complains again.
